extensions/git/src/repository.ts

```diff
@@ -866,9 +866,12 @@ export class Repository implements Disposable {
 
 	async getInputTemplate(): Promise<string> {
 		const mergeMessage = await this.repository.getMergeMessage();
+		const squashMessage = await this.repository.getSquashMessage();
```

Comment:
Should we parallelize these two?

Reply:
Yup, that makes complete sense to me. I've changed the variable to be `commitMessage` in my latest commit, and it now checks for `getMergeMessage()` OR `getSquashMessage` considering we'll never use them both at the same time.